data/yam/agama/auto/lib/questions.libsonnet

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
in fact following the same logic I would call this library `answers.libsonnet` because it would be more logical and meaningful.